### PR TITLE
Add Delaware.gov

### PR DESCRIPTION
**Pull request by Yang-Nankai**
Delaware.gov is the official website of the state of Delaware, providing access to government services, information, and resources for residents, businesses, and visitors. It includes tools for managing state taxes, renewing licenses, and accessing public records. The website offers 2FA for the account login verification process. The website is ranked 26,012 on [Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=delaware.gov). The website provides documentation on how to turn on 2FA.

### CODE REVIEW

File: entries/d/delaware.gov.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Delaware.gov": {
+    "domain": "delaware.gov",
+    "tfa": [
+      "call",
+      "sms"
+    ],
+    "documentation": "https://webfiles.dti.delaware.gov/pdfs/Setting-Up-Your-MFA.pdf6",
```
Comment (Carlgo11) [inline entries/d/delaware.gov.json]: :wink: 
```suggestion
    "documentation": "https://webfiles.dti.delaware.gov/pdfs/Setting-Up-Your-MFA.pdf",
```

File: entries/d/delaware.gov.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "Delaware.gov": {
+    "domain": "delaware.gov",
+    "tfa": [
+      "call",
+      "sms"
```
Comment (Carlgo11) [inline entries/d/delaware.gov.json]: From this super low res screenshot, it looks like they also support Okta Verify as an authentication method.
image

```suggestion
      "sms",
      "custom-software"
    ],
    "custom-software": [
      "Okta Verify"  
```

**Review by Carlgo11 — DISMISSED** [review]

File: entries/d/delaware.gov.json (hunk 2 above)
Comment (Yang-Nankai) [reply entries/d/delaware.gov.json]: Wow, thanks for the correction, I thought Okta was outside the scope of our 2FA methods! ❤️  @Carlgo11

File: entries/d/delaware.gov.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      "sms",
+      "custom-software"
+    ],
+    "custom-software": [
+      "Okta Verify"  
+    ],
+    "documentation": "https://webfiles.dti.delaware.gov/pdfs/Setting-Up-Your-MFA.pdf",
+    "categories": [
+      "government"
+    ]
```
Comment (hkamran80) [inline entries/d/delaware.gov.json:18]:
```suggestion
    ],
    "regions": [
      "us"
    ]
```

**Review by hkamran80 — DISMISSED** [review]

**Review by fletcherdodd — APPROVED** [review]

**Pull request merged**